boards/xtensa/esp32s2_devkitc/support/openocd.cfg

```diff
@@ -0,0 +1,7 @@
+set ESP_RTOS none
+set ESP32_ONLYCPU 1
+
+# Source the JTAG interface configuration file
+source [find interface/esp_usb_jtag.cfg]
```

Comment:
The ESP32-S2 doesn't have a built-in JTAG interface

Reply:
It was done